statusHandler/presenter/common.go

```diff
@@ -94,7 +84,17 @@ func (psh *PresenterStatusHandler) computeRoundsPerHourAccordingToHitRate() floa
 
 func (psh *PresenterStatusHandler) computeRewardsInErd() *big.Float {
 	rewardsValue := psh.getBigIntFromStringMetric(core.MetricRewardsValue)
-	denominationCoefficient := psh.getBigFloatFromStringMetric(core.MetricDenominationCoefficient)
+	denomination := psh.getFromCacheAsUint64(core.MetricDenomination)
+	denominationCoefficientStr := "0."
```

Comment:
I don't think that this works as expected. If denomination would be 0, then the result of denominationCoefficientStr would be 0.1 instead of 1. If denomination would be 1 then the result of denominationCoefficientStr would be 0.01 instead of 0.1. So every time it would be wrong with exactly one order of magnitude. I suggest to replace the code between lines 88-96 with the below proposed which is much simplified and I think it works correct.

```
	denominationCoefficientFloat := 1.0
	if denomination > 0 {
		denominationCoefficientFloat /= float64(denomination)
	}
	
	denominationCoefficient := big.NewFloat(denominationCoefficientFloat)
```

Reply:
you're right. fixed